azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -64,7 +64,7 @@
   private static final String MAIL_CREATOR = "mailCreator";
   private static final String MEMORY_CHECK = "memoryCheck";
   // Flow restartability check
-  private static final String EXECUTION_RETRY = "executionRetryByAzkaban";
+  private static final String EXECUTION_RETRY_BY_AZKABAN = "executionRetryByAzkaban";
```

Comment:
You can remove this field and use the one defined in Constants.

Reply:
Sounds good! Done.